content/en/docs/general-usage/sdv-introduction/_index.md

```diff
@@ -77,18 +76,19 @@ A good starting point to figure out what signals are available is the website of
 
 ![digital.auto Vehicle API](digitalauto-vehicle-api.png)
 
-In the next step, one can ideate on new applications, start coding, and get fast feedback using the prototyping feature of the playground.
-Once one is confident with the initial version, one can continue the development journey with the help of [Eclipse Velocitas](https://eclipse.dev/velocitas/), 
+In the next step, we can ideate on new applications, start coding, and get fast feedback using the prototyping feature of the playground.
```

Comment:
I would use 'you' here instead. This way we would differentiate were the contributors (we) created something (e.g., the architecture) and where the reader can get active.

Reply:
You are right. I changed the "we"'s I have introduced with "you"